go.sum

```diff
@@ -1,3 +1,4 @@
+github.com/discentem/gorilla v1.0.0-beta.3 h1:O+HiFU+tTbTY6nu9W0QfSQlZt19C1TelRUmNDCqDhSI=
```

Comment:
I think this is because `go` gets confused about the fork.

I am not sure what the "right" solution is. [This stackoverflow answer](https://stackoverflow.com/a/29161911) has a good solution, but that might be a headache to do at this point. Maybe just delete the new line 😬?

Reply:
Manually removed.